cfme/fixtures/vm.py

```diff
@@ -113,11 +127,53 @@ def create_vm(setup_provider, request, provider):
     if request.param:
         template_type = request.param
     else:
-        pytest.error('Any appropriate Template was not passed to the fixture.')
+        pytest.error('A VM/Instance template type was not passed to the fixture.')
+
     vm_name = _get_vm_name(request)
     return _create_vm(request, template_type, provider, vm_name)
 
 
+@pytest.fixture(scope="module")
+def create_vms_modscope(setup_provider_modscope, request, provider):
+    """Create one or more VMs/Instances."""
+    template_type = None
+    param = request.param
+
+    if param and isinstance(param, dict):
```

Comment:
As @mshriver pointed out on other PR, if the param is not present, or not a dict, we will get unboundLocal error on L150 below for num_vms. please address it. Thanks.

Reply:
Updated.